django-pagination/terms/templates/terms/infinite_scrolling.html

```diff
@@ -0,0 +1,59 @@
+{% extends "terms/base.html" %}
+
+{% block pagination %}
+
+<style>
+    #loading {
+        transition: opacity 1s ease-out;
+        opacity: 1;
+    }
+</style>
+
+<div id="loading">Loading...</div>
+
+<script>
+    class ScrollMorePaginator {
+        constructor(perPage) {
+            this.perPage = perPage
+            this.pageIndex = 0
+            this.elements = document.querySelectorAll("pre")
+            this.loader = document.querySelector("#loading")
+            this.options = {
+                root: null,
+                rootMargin: "0px",
+                threshold: 0.25
+            }
+            this.loadMore()
+            this.watchIntersection()
+        }
+
+        onIntersect() {
+            this.pageIndex++
+            this.loadMore()
+        }
+
+        watchIntersection() {
+            document.addEventListener("DOMContentLoaded", () => {
+                let observer = new IntersectionObserver(this.onIntersect.bind(this),
```

Comment:
It's a common "best practice" to use `const` as much as possible and only use `let` when necessary.